helper_functions.py

```diff
@@ -0,0 +1,29 @@
+import numpy as np
```

Comment:
I think it is starting to make sense to split the psplot file up into multiple files. It helps with clearity and I see that you import this file into the other, and in this file noting is imported from psplot so very happy with that 👍